src/handlers/motions/motionCreated/helpers.ts

```diff
@@ -80,6 +80,7 @@ const getMotionData = async (
     motionDomainId: domainId.toString(),
     stakerRewards: [],
     isFinalized: false,
+    events: [{ name: 'MotionCreated', initiatorAddress: '', vote: '', amount: '' }],
```

Comment:
`initiatorAddress` should have a default value of the null address

Reply:
Also, why are vote and amount present here? Neither are relevant to the motion being created.

Reply:
I am using the creator now which I believe is the same as msgSender.